src/input/mod.rs

```diff
@@ -152,22 +150,30 @@ impl<'a> InputFile<'a> {
 
 
     /// Reads image data into the given FrameBufferMut.
+    ///
+    /// The passed FrameBufferMut must match the image's resolution exactly, and
```

Comment:
On third thought, I forgot my own code, and they don't panic, they return an error.  Nevertheless, we should probably still document it on the methods.